frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -101,8 +104,8 @@ const getResourceName = (networkType, typeParamsData): string => {
     : `openshift.io/${_.get(typeParamsData, 'resourceName.value', '')}`;
 };
 
-const generateNADName = (name: string): string => {
-  return `${name}-${uniqueNamesGenerator({
+const generateNADName = (): string => {
+  return `network-${uniqueNamesGenerator({
```

Comment:
Can it be `nad-${uniqueNamesGenerator... ?
Or `netattdef-${uniqueNamesGenerator... ?
Common word "network" with randomized suffix is difficult to test for